protocol/app/upgrades/v7.0.0/upgrade.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 const (
-	// Each megavault share is worth 1 USDC.
-	QUOTE_QUANTUMS_PER_MEGAVAULT_SHARE = 1_000_000
+	// Each megavault share is worth 0.01 USDC.
+	QUOTE_QUANTUMS_PER_MEGAVAULT_SHARE = 10_000
```

Comment:
nit: 1000? If we are in the sub-cent range, it will probably mean less rounding errors that can be seen.